core/jreleaser-model/src/main/java/org/jreleaser/model/validation/UploadersValidator.java

```diff
@@ -43,14 +44,16 @@ public static void validateUploaders(JReleaserContext context, JReleaserContext.
         validateS3(context, mode, errors);
         validateScpUploader(context, mode, errors);
         validateSftpUploader(context, mode, errors);
+        validateAzureArtifacts(context, mode, errors);
 
         if (mode.validateConfig() && !upload.isEnabledSet()) {
             upload.setEnabled(!upload.getActiveArtifactories().isEmpty() ||
                 !upload.getActiveFtps().isEmpty() ||
                 !upload.getActiveHttps().isEmpty() ||
                 !upload.getActiveS3s().isEmpty() ||
                 !upload.getActiveScps().isEmpty() ||
-                !upload.getActiveSftps().isEmpty());
+                !upload.getActiveSftps().isEmpty() ||
+                !upload.getActiveAzureArtifactses().isEmpty());
```

Comment:
This method name is a bit weird as there's a double plural but I guess we can live with it for now.